src/common/servlist.c

```diff
@@ -139,6 +140,7 @@ static const struct defaultserver def[] =
 	{0,			"irc.inet.tele.dk"},
 
 	{"ElectroCode", 0, 0, 0, LOGIN_SASL, 0, TRUE},
+	{0,			"irc.electrocode.net/6667"},
```

Comment:
No.. Remove the ports from both of your entries and remove the duplicated servers.